### PR TITLE
[JKT-2020] Initial Commit DevOpsDays Jakarta 2020

### PR DESCRIPTION
**Pull request by mademulia**
[JKT-2020] Initial event commit for DevOpsDays Jakarta 2020

### CODE REVIEW

File: data/events/2020-jakarta.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2020-03-10T00:00:00+07:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2020-03-12T23:59:59+07:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
```
Comment (MikeRosTX) [inline data/events/2020-jakarta.yml]: We do not allow publishing a date if there is not a contract signed with the location venue.

File: data/events/2020-jakarta.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Jakarta" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
```
Comment (MikeRosTX) [inline data/events/2020-jakarta.yml]: If you do have a signed contract with the venue, remove the hash (#) to add location navbar, and add location coordinate plus address.

File: data/events/2020-jakarta.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+ # - name: registration
+  - name: program
```
Comment (MikeRosTX) [inline data/events/2020-jakarta.yml]: Remark out Program navbar.

File: data/events/2020-jakarta.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    image: "made_indrajaya.jpeg"
+  - name: "Mia Fauziah"
+    linkedin: "https://www.linkedin.com/in/mia-fauziah-856750170/"
+    image: "mia_fauziah.jpg"
+  - name: "Stefan Streichsbier"
+    github: "streichsbaer"
+    twitter: "s_streichsbier"
+    linkedin: "https://www.linkedin.com/in/sstreichsbier"
+    image: "stefan_streichsbier.jpg"
+organizer_email: "admin@devopsdaysjkt.org" # Put your organizer email address here
```
Comment (MikeRosTX) [inline data/events/2020-jakarta.yml]: We do not allow other email domains to be published. This needs to be **_jakarta@devopsdays.org_**.

**Comment by mademulia** [comment]
Hi @MikeRosTX, just updated the file as advised. Thank you for reviewing

**Pull request merged**